app/views/course/users/_list.html.slim

```diff
@@ -0,0 +1,21 @@
+div.table-fluid
+  table.table.table-striped
+    thead
+      tr
+        th = t('.name')
+        th = t('.email')
+        th = t('.phantom')
+        th
+
+    tbody
+      - users.each do |user|
```

Comment:
Shall we just implement a approve_multiple and reject_multiple instead of a submit button for each user ?